regression/cbmc/r_w_ok_inconsistent_invalid/main.c

```diff
@@ -0,0 +1,24 @@
+#include <assert.h>
+#include <stdlib.h>
+
+char *p;
+
+void test_global_pointer()
+{
+  // special value of invalid pointer
```

Comment:
:question: Why is it special?

Reply:
Internally,`1 << 56` (when `sizeof(char *) = 8` and `object_bits = 8`) is a special value which at various points is used to denote an invalid pointer (i.e., the object with value 1). `__CPROVER_is_invalid_pointer()` just checks for this value.

Reply:
This explanation should therefore be in the code, perhaps as:
```
size_t object_bits = 8;
// special value of invalid pointer see __CPROVER_is_invalid_pointer
char *p = (size_t)1 << (sizeof(char *) * object_bits - object_bits);
```

Reply:
Added